for (;;){
   try{ 
	throw b
     const a=new a()
   }catch(err){
     
   }finally{x--}
}